File: python/test/pdfkit/hello_world.py
import pdfkit

html_content = """
<html>
  <head>
    <title>Hello, World!</title>
  </head>
  <body>
    <h1>Hello, World!</h1>
  </body>
</html>
"""

pdfkit.from_string(html_content, "hello_world.pdf")

print("PDF created successfully!")

File: archive/cs/hello_world.cs
using iText.Kernel.Pdf;
using iText.Layout;
using iText.Layout.Element;
using System.IO;

class Program
{
    static void Main(string[] args)
    {
        string filename = "hello_world.pdf";

        using (var writer = new PdfWriter(filename))
        using (var pdfDoc = new PdfDocument(writer))
        using (var document = new Document(pdfDoc))
        {
            document.Add(new Paragraph("Hello World!"));
        }

        Console.WriteLine($"PDF created successfully: {filename}");
    }
}
